scripts/projector.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# failfast
+set -e
+
+# Define these 2 optional variables if you want to customize the default installation,
+# then copy-paste the remainder of the script:
+# - the name of the IDE to install (defaults to "IntelliJ IDEA Community Edition 2020.3.4")
+# Note: to get the list of available IDEs, launch `projector find`
+# projector_ide_name="IntelliJ IDEA Community Edition 2020.3.4"
+# - the password to protect access to projector (defaults to no password)
+# projector_password="class-specific-random-password"
+
+projector_ide_name=${projector_ide_name:-IntelliJ IDEA Community Edition 2020.3.4}  # Latest projector-tested IDE
```

Comment:
Works perfectly fine with 2021.1.2 version

Reply:
I preferred to stay with the "tested" channel by default.